editor/store/effects.js

```diff
@@ -452,4 +452,10 @@ export default {
 		const message = spokenMessage || content;
 		speak( message, 'assertive' );
 	},
+	INITIALIZE_META_BOX_STATE( action ) {
+		// Allow toggeling metaboxes panels
+		if ( some( action.metaBoxes ) ) {
```

Comment:
Minor: `some` is reasonable enough; `isEmpty` is maybe a bit more accurate to what you're testing, particularly since you're not providing a predicate for `some`.

Reply:
Actually, isEmpty doesn't work here because it's an object like this `{ side: true, normal: false }`. isEmpty always returns false in this case.